client/packages/common/src/ui/components/inputs/SearchBar/SearchBar.tsx

```diff
@@ -10,17 +10,25 @@ import { useTranslation } from '@common/intl';
 interface SearchBarProps {
   value: string;
   onChange: (value: string) => void;
+  onClear?: () => void;
+  onSearchIconClick?: () => void;
+  searchIconButtonLabel?: string;
   placeholder: string;
   isLoading?: boolean;
   debounceTime?: number;
   expandOnFocus?: boolean;
+  autoFocus?: boolean;
```

Comment:
thought autofocus would be nice, otherwise have to click twice. Also added onclear/on icon click, to be able to close the search both when clearing the value, or when clicking the search icon... let me know how it feels to use?

Reply:
When i did the item on row, I added the search to VEN (and to pop up left to right), making the icon no accessible, so in that one I added `alwaysClear` prop (should really be alwaysShowClear)